index.js

```diff
@@ -54,12 +54,12 @@ module.exports.output = function (url, opts, cb) {
     console.log(data);
 
     try {
-      output(handleOpts(url, opts), data);
+      output(handleOpts(url, opts), data, function() {
+        cb();
+      });
     } catch (err) {
       cb(err);
       return;
```

Comment:
actually, with https://github.com/addyosmani/psi/pull/59/files#r40241237 the try/catch can be removed